packages/web-components/src/components/masthead/top-nav.ts

```diff
@@ -430,12 +430,7 @@ class DDSTopNav extends StableSelectorMixin(HostListenerMixin(BXHeaderNav)) {
                   <div class="${prefix}--header__nav-content" style="right: -${currentScrollPosition}px">
                     <nav part="nav" class="${prefix}--header__nav">
                       <div class="${prefix}--sub-content-right"></div>
-                      <ul
```

Comment:
Now that this is removed, new a11y errors are introduced since `top-nav-menu` still contains items with `role=menuitem`. Perhaps we can remove via `connectedCallback`?

![image](https://user-images.githubusercontent.com/909118/135621530-bca2ec78-6340-462a-b3f3-6046eea1a4d8.png)


https://github.com/carbon-design-system/carbon-web-components/blob/01b289238cd9a8edceaa3ad1ee548cb6693a5da8/src/components/ui-shell/header-menu.ts#L121